backend/src/main/java/com/example/backend/data_access/stop/StopDAO.java

```diff
@@ -44,7 +44,7 @@ public HashSet<String> getStopTagsByRouteTag(String routeTag) {
             for (int i = 0; i < nodeList.getLength(); i++) {
                 Element element = (Element) nodeList.item(i);
 
-                stopTags.add(element.getAttribute("tag"));
+                stopTags.add(element.getAttribute("tag").split("_")[0]);
```

Comment:
Do we still need to make an API call for this function? Since all other functions like getRouteByRouteTag already only needs to read from the CSV file, this function can be implemented by just reading from the CSV right?

Reply:
We don't, the only reason it hasn't been updated is because it currently works and only costs 1 API call. I've gone ahead and updated it.